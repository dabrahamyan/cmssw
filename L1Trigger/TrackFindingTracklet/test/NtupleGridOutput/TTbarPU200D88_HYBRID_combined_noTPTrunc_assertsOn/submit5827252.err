************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 125X_mcRun4_realistic_v5.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
03-Sep-2023 13:32:18 CEST  Initiating request to open file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/de3299ef-1399-4acb-a2f2-40676219dd8b.root
03-Sep-2023 13:32:28 CEST  Successfully opened file root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/de3299ef-1399-4acb-a2f2-40676219dd8b.root
%MSG-w XrdAdaptorLvl3:  AfterFile 03-Sep-2023 13:32:41 CEST pre-events
Deactivating heplnx229.pp.rl.ac.uk from active sources because its quality (691) is higher than 260 and 4 times larger than the other active server heplnx228.pp.rl.ac.uk (18) 
%MSG
L1TrackNtupleMaker::beginJob
Begin processing the 1st record. Run 1, Event 9001, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:01.484 CEST
Begin processing the 2nd record. Run 1, Event 9003, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:05.668 CEST
Begin processing the 3rd record. Run 1, Event 9002, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:09.209 CEST
Begin processing the 4th record. Run 1, Event 9004, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:13.425 CEST
Begin processing the 5th record. Run 1, Event 9005, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:16.876 CEST
Begin processing the 6th record. Run 1, Event 9006, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:21.000 CEST
Begin processing the 7th record. Run 1, Event 9007, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:24.564 CEST
Begin processing the 8th record. Run 1, Event 9008, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:28.139 CEST
Begin processing the 9th record. Run 1, Event 9009, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:31.065 CEST
Begin processing the 10th record. Run 1, Event 9010, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:34.245 CEST
Begin processing the 11th record. Run 1, Event 9011, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:37.172 CEST
Begin processing the 12th record. Run 1, Event 9012, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:40.434 CEST
Begin processing the 13th record. Run 1, Event 9013, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:43.855 CEST
Begin processing the 14th record. Run 1, Event 9014, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:46.984 CEST
Begin processing the 15th record. Run 1, Event 9015, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:50.554 CEST
Begin processing the 16th record. Run 1, Event 9016, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:54.939 CEST
Begin processing the 17th record. Run 1, Event 9017, LumiSection 91 on stream 0 at 03-Sep-2023 13:33:58.321 CEST
Begin processing the 18th record. Run 1, Event 9019, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:02.369 CEST
Begin processing the 19th record. Run 1, Event 9018, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:05.601 CEST
Begin processing the 20th record. Run 1, Event 9020, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:09.274 CEST
Begin processing the 21st record. Run 1, Event 9021, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:12.771 CEST
Begin processing the 22nd record. Run 1, Event 9022, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:15.798 CEST
Begin processing the 23rd record. Run 1, Event 9023, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:18.906 CEST
Begin processing the 24th record. Run 1, Event 9024, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:22.229 CEST
Begin processing the 25th record. Run 1, Event 9025, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:25.207 CEST
Begin processing the 26th record. Run 1, Event 9026, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:30.035 CEST
Begin processing the 27th record. Run 1, Event 9027, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:33.692 CEST
Begin processing the 28th record. Run 1, Event 9029, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:37.654 CEST
Begin processing the 29th record. Run 1, Event 9028, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:40.670 CEST
Begin processing the 30th record. Run 1, Event 9031, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:43.692 CEST
Begin processing the 31st record. Run 1, Event 9030, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:47.147 CEST
Begin processing the 32nd record. Run 1, Event 9032, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:50.478 CEST
Begin processing the 33rd record. Run 1, Event 9033, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:53.215 CEST
Begin processing the 34th record. Run 1, Event 9035, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:56.311 CEST
Begin processing the 35th record. Run 1, Event 9034, LumiSection 91 on stream 0 at 03-Sep-2023 13:34:59.071 CEST
Begin processing the 36th record. Run 1, Event 9036, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:02.025 CEST
Begin processing the 37th record. Run 1, Event 9038, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:05.382 CEST
Begin processing the 38th record. Run 1, Event 9039, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:09.193 CEST
Begin processing the 39th record. Run 1, Event 9037, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:12.996 CEST
Begin processing the 40th record. Run 1, Event 9041, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:16.053 CEST
Begin processing the 41st record. Run 1, Event 9040, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:19.597 CEST
Begin processing the 42nd record. Run 1, Event 9042, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:23.240 CEST
Begin processing the 43rd record. Run 1, Event 9044, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:26.349 CEST
Begin processing the 44th record. Run 1, Event 9043, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:29.887 CEST
Begin processing the 45th record. Run 1, Event 9046, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:33.559 CEST
Begin processing the 46th record. Run 1, Event 9045, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:37.291 CEST
Begin processing the 47th record. Run 1, Event 9047, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:40.204 CEST
Begin processing the 48th record. Run 1, Event 9048, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:43.879 CEST
Begin processing the 49th record. Run 1, Event 9049, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:47.236 CEST
Begin processing the 50th record. Run 1, Event 9050, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:51.093 CEST
Begin processing the 51st record. Run 1, Event 9052, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:54.919 CEST
Begin processing the 52nd record. Run 1, Event 9051, LumiSection 91 on stream 0 at 03-Sep-2023 13:35:57.878 CEST
Begin processing the 53rd record. Run 1, Event 9053, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:00.693 CEST
Begin processing the 54th record. Run 1, Event 9054, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:03.360 CEST
Begin processing the 55th record. Run 1, Event 9055, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:06.698 CEST
Begin processing the 56th record. Run 1, Event 9056, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:10.160 CEST
Begin processing the 57th record. Run 1, Event 9057, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:14.046 CEST
Begin processing the 58th record. Run 1, Event 9059, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:17.706 CEST
Begin processing the 59th record. Run 1, Event 9058, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:20.922 CEST
Begin processing the 60th record. Run 1, Event 9060, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:24.882 CEST
Begin processing the 61st record. Run 1, Event 9061, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:29.340 CEST
Begin processing the 62nd record. Run 1, Event 9062, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:32.885 CEST
Begin processing the 63rd record. Run 1, Event 9063, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:36.288 CEST
Begin processing the 64th record. Run 1, Event 9064, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:39.039 CEST
Begin processing the 65th record. Run 1, Event 9065, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:42.145 CEST
Begin processing the 66th record. Run 1, Event 9066, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:45.135 CEST
Begin processing the 67th record. Run 1, Event 9068, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:48.125 CEST
Begin processing the 68th record. Run 1, Event 9067, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:51.331 CEST
Begin processing the 69th record. Run 1, Event 9071, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:54.027 CEST
Begin processing the 70th record. Run 1, Event 9069, LumiSection 91 on stream 0 at 03-Sep-2023 13:36:57.443 CEST
Begin processing the 71st record. Run 1, Event 9070, LumiSection 91 on stream 0 at 03-Sep-2023 13:37:00.910 CEST
Begin processing the 72nd record. Run 1, Event 9073, LumiSection 91 on stream 0 at 03-Sep-2023 13:37:04.427 CEST
Begin processing the 73rd record. Run 1, Event 9074, LumiSection 91 on stream 0 at 03-Sep-2023 13:37:07.952 CEST
Begin processing the 74th record. Run 1, Event 9072, LumiSection 91 on stream 0 at 03-Sep-2023 13:37:11.448 CEST
Will break on number of tracklets in TP_L1L2D
cmsRun: /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/src/L1Trigger/TrackFindingTracklet/src/TrackletProcessor.cc:345: void trklet::TrackletProcessor::execute(unsigned int, double, double): Assertion `0' failed.


A fatal system signal has occurred: abort signal
The following is the call stack containing the origin of the signal.


Current Modules:

Module: L1FPGATrackProducer:l1tTTTracksFromTrackletEmulation (crashed)

A fatal system signal has occurred: abort signal
condor_exec.exe: line 7:   155 Aborted                 (core dumped) cmsRun L1TrackNtupleMaker_cfg_grid_HYBRID_trkJets.py inputFiles=root://cms-xrd-global.cern.ch///store/mc/CMSSW_12_6_0/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_125X_mcRun4_realistic_v5_2026D88PU200RV183v2-v1/30000/de3299ef-1399-4acb-a2f2-40676219dd8b.root outputFile=/eos/user/d/dabraham/L1NtupleTrackExamples/TTbarPU200D88_HYBRID_combined_noTPTrunc_assertsOn_10.root maxEvents=10000
