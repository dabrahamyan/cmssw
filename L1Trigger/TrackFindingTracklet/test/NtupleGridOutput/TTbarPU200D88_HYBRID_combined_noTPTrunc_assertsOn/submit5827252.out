using geometry D88 (tilted)
Sun Sep  3 13:37:14 CEST 2023
Thread 7 (Thread 0x2b0dea04d700 (LWP 184) "cmsRun"):
#0  0x00002b0db6526b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b0db6526bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b0db6526c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b0dbd00a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b0dbd00a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b0de9e4c700 (LWP 183) "cmsRun"):
#0  0x00002b0db6526b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b0db6526bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b0db6526c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b0dbd00a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b0dbd00a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b0de9c4b700 (LWP 182) "cmsRun"):
#0  0x00002b0db6526b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b0db6526bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b0db6526c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b0dbd00a576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b0dbd00a639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b0de9a4a700 (LWP 181) "cmsRun"):
#0  0x00002b0db6527e9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b0dbcd4710f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b0dbcf782d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002b0dbcf78429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b0de9849700 (LWP 180) "cmsRun"):
#0  0x00002b0db68340e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b0dbcd415c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b0dbcd3dd4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002b0dbcd46818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b0dd9f6e700 (LWP 173) "cmsRun"):
#0  0x00002b0db65281d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b0dbd667ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b0dbd6686ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b0db60fcf90 in std::execute_native_thread_routine (__p=0x2b0dc9651e30) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b0db6520ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b0db6833b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b0db88b6e40 (LWP 155) "cmsRun"):
#0  0x00002b0db6828ddd in poll () from /lib64/libc.so.6
#1  0x00002b0dbd667dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b0dbd66879c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b0dbd66b24b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b0db676b387 in raise () from /lib64/libc.so.6
#6  0x00002b0db676ca78 in abort () from /lib64/libc.so.6
#7  0x00002b0db67641a6 in __assert_fail_base () from /lib64/libc.so.6
#8  0x00002b0db6764252 in __assert_fail () from /lib64/libc.so.6
#9  0x00002b0de7460331 in trklet::TrackletProcessor::execute(unsigned int, double, double) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#10 0x00002b0de73dcb59 in trklet::Sector::executeTP() () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#11 0x00002b0de744d263 in trklet::TrackletEventProcessor::event(trklet::SLHCEvent&, std::vector<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >, std::allocator<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > >&, std::vector<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> >, std::allocator<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> > > >&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#12 0x00002b0de7287a74 in L1FPGATrackProducer::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#13 0x00002b0db41439ae in edm::one::EDProducerBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#14 0x00002b0db412d23f in edm::WorkerT<edm::one::EDProducerBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#15 0x00002b0db407db25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#16 0x00002b0db407de13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002b0db4080b5a in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002b0db4080dd1 in edm::SerialTaskQueue::QueuedTask<edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(tbb::detail::d1::task_group&, edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x00002b0db3e18175 in tbb::detail::d1::function_task<edm::SerialTaskQueue::spawn(edm::SerialTaskQueue::TaskBase&)::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#20 0x00002b0db5913bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2b0dba046c00, this=0x2b0dba03be00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#21 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2b0dba03be00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#22 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#23 0x00002b0db3ff822d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#24 0x00002b0db3fe0310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#25 0x00002b0db3fecda1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#26 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#27 0x00002b0db59020eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#28 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#29 0x000000000040971c in main ()
